### DISCUSSION TITLE
Vantage like for Linux

### DISCUSSION THREAD

**Discussion by Nosnahc** [4JX/L5P-Keyboard-RGB#105] [Ideas]
Hi,
I wanted to thank you for your great work.
I can change my keyboard color without Windows and it's great!

I was wondering if another tab with features similar to Vantage wouldn't be a real plus for this program?
By this I mean a tab with sliders that would allow you to switch the battery to Conservation mode or to mute the microphone, like on [this Git](https://github.com/niizam/vantage) but better done?
I am aware that this is quite far from the initial subject but I thought so I share.

Thanks again for your great work!!!

**Comment by 4JX** [comment]
You may try https://github.com/johnfanv2/LenovoLegionLinux for that, I currently consider adding other stuff to enter into the scope of feature creep.

**Comment by HyperN00B** [comment]
Performance profiles could sorta easily be handled by power-profiles-daemon.
Also maybe you could add a customizable hotkey to increase/decrease brightness (i can recommend Fn+Up, Fn+Down as on ROG Zephyrus) cause even Windows lacks that feature
